themes/dohmh/layouts/data-explorer/single.html

```diff
@@ -2796,19 +2802,25 @@ <h6>${secondaryIndicatorName} - ${secondaryMeasure}</h6>
         let allData = aq.from(fullDataTableObjects)
             .groupby("Time", "GeoType", "GeoID", "GeoRank", "Geography")
             .pivot("MeasurementDisplay", "DisplayCI")
-            .relocate(["Time", "GeoType", "GeoID", "GeoRank"], { before: 0 }) 
+            .relocate(["Time", "GeoType", "GeoID", "GeoRank"], { before: 0 })
 
         let downloadTableCSV = allData.toCSV();
 
         // Data URI
         let csvData = 'data:application/csv;charset=utf-8,' + encodeURIComponent(downloadTableCSV);
         let hiddenElement = document.createElement('a');
-        
+
         hiddenElement.href = csvData;
         hiddenElement.target = '_blank';
         hiddenElement.download = 'NYC EH Data Portal - ' + indicatorName + " (full)" + '.csv';
         hiddenElement.click();
 
+        gtag('event', 'file_download', {
```

Comment:
@mmontesanonyc this one was easy because of the `hiddenElement.download` variable

Reply:
Could you just use the same JS variable, `indicatorName`, for the other download? Something like:
```
'file_name': indicatorName + ' table view.'
```

Reply:
@mmontesanonyc yes, that did the trick. thank you. I just pushed up code for this, so if looks good, please go ahead and merge.